apps/web/src/pages/SearchRequests/RequestConfirmationPage/components/RequestConfirmationPrintDocument.tsx

```diff
@@ -61,8 +61,8 @@ const RequestConfirmationPrintDocument = forwardRef<
                 id: "3rbRfI",
                 description:
                   "Description of when the user should expect a response to their request",
-                // eslint-disable-next-line formatjs/no-literal-string-in-jsx
-              })}{" "}
+              })}
+              <span className="mr-1"></span>
```

Comment:
I doubt it, but this odd grammar could cause issues with some screen readers. I wonder, why did we make this two separate messages? It seems like it is all a single message 🤔 

Result would be "...10 business days.If you have..." where there is not space between the period and the following message.

Reply:
[8d3f5aa](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14201/commits/8d3f5aab0dfb841b4914db77b1bb28c8f3bb2511)